src/relay/op/tensor/elemwise.cc

```diff
@@ -106,6 +182,63 @@ RELAY_REGISTER_OP("subtract")
   // input2: Tensor[dtype, s2]
   // output: Tensor[dtype, broadcast(s1, s2)]
 
+// Division
+TVM_REGISTER_API("relay.op._make.divide")
+  .set_body_typed<Expr(Expr, Expr)>([](Expr lhs, Expr rhs) {
+      static const Op& op = Op::Get("divide");
+    return CallNode::make(op, {lhs, rhs}, Attrs(), {});
+  });
+
+RELAY_REGISTER_OP("divide")
```

Comment:
since BINARY_BCAST operator is soo common, create a macro for it just like unary operations.